tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -0,0 +1,204 @@
+			Expect(framework.AsKubeAdmin.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+			Expect(framework.SandboxController.DeleteUserSignup(framework.UserName)).NotTo(BeFalse())
+		}
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that a PipelineRun is created in dev namespace.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, componentName)
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that in managed namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, "release")
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies a PipelineRun started in managed namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("validate that in result of task create-pyxis-image and id and succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.AsKubeAdmin.TektonController.ListAllPipelineRuns(managedNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				pr, err := kubeController.Tektonctrl.GetPipelineRun(prList.Items[0].Name, managedNamespace)
+				Expect(err).NotTo(HaveOccurred())
+
+				tr, err := kubeController.GetTaskRunStatus(pr, "create-pyxis-image")
+				Expect(err).NotTo(HaveOccurred())
+
+				return strings.Contains(string(tr.Status.Status.Conditions[0].Status), "True") &&
+					strings.Contains(string(tr.Status.TaskRunResults[0].Name), "containerImageIDs")
```

Comment:
Yes I can see and ID with a value, see below example from log of task **create-pyxis-image** :

STEP-CREATE-PYXIS-IMAGE

2023-03-28 11:20:13,744 [pyxis] INFO Image with given docker_image_digest doesn't exist yet
2023-03-28 11:20:13,745 [pyxis] INFO Creating new container image
2023-03-28 11:20:14,537 [pyxis] INFO The image id is: 6422cd6e620fa8b1d095ba4e

Reply:
Yes. That is when it works. I am saying even when it doesn't, it may say
```
2023-03-28 11:20:13,744 [pyxis] INFO Image with given docker_image_digest doesn't exist yet
2023-03-28 11:20:13,745 [pyxis] INFO Creating new container image
2023-03-28 11:20:14,537 [pyxis] INFO The image id is: 
```
I am not sure. That is why I am asking to check the result value for `The image id is: xxxxxxx` instead of just checking that there is a result named `containerImageIDs`